airflow/utils/db_cleanup.py

```diff
@@ -182,7 +182,7 @@ def _do_delete(*, query, orm_model, skip_archive, session):
     metadata = reflect_tables([orm_model.name, target_table_name], session)
     source_table = metadata.tables[orm_model.name]
     target_table = metadata.tables[target_table_name]
-    logger.debug("rows moved; purging from %s", source_table.name)
+    logger.debug("rows moved; purging from %s", str(source_table.name))
```

Comment:
Are we sure that we want to include this fix in this PR? Because it seems a bit hacky here, and just hide actual problem